resources/definitions/fdmprinter.def.json

```diff
@@ -5855,7 +5855,7 @@
                     "description": "The minimum size of a line segment after slicing. If you increase this, the mesh will have a lower resolution. This may allow the printer to keep up with the speed it has to process g-code and will increase slice speed by removing details of the mesh that it can't process anyway.",
                     "type": "float",
                     "unit": "mm",
-                    "default_value": 0.01,
+                    "default_value": 0.25,
                     "minimum_value": "0.001",
```

Comment:
We will need to modify also the `minimum_value`, `minimum_value_warning` and `maximum_value_warning` accordingly to the new value.

Reply:
Yeah, I saw this also when Tom was using it.